package.json

```diff
@@ -66,11 +66,8 @@
     ],
     "topicSeparator": " ",
     "topics": {
-      "hello": {
-        "description": "Say hello to the world and others"
-      },
-      "runtime": {
-        "description": "Create a new Vehicle App"
+      "cache": {
```

Comment:
You could maybe also add the other command topics here

Reply:
Topics are only relevant for subcommands where the "parent" command has no index.ts by itself. This will provide the description when calling `velocitas cache` without subcommand. I'd say lets only use it when we need it.